src/Module/Media/Photo/Upload.php

```diff
@@ -168,9 +169,9 @@ protected function post(array $request = [])
 		$width  = $image->getWidth();
 		$height = $image->getHeight();
 
-		$maximagesize = $this->config->get('system', 'maximagesize');
+		$maximagesize = Strings::getBytesFromShorthand(DI::config()->get('system', 'maximagesize'));
```

Comment:
Please keep the class variable call, we're trying to move away from the `DI` class that is a transitional compromise between the all-static classes we had before and the Dependency Injection model we are striving towards.
```suggestion
		$maximagesize = Strings::getBytesFromShorthand($this->config->get('system', 'maximagesize'));
```

Reply:
Ohh oh okay, but here in `Upload.php` is the only place yet where we can do it "the modern way"? Because I used DI anywhere else :-/

Reply:
Converting to DI requires additional changes that aren't really in the scope of this PR. In this class the `config` property is available so you should use it. In other classes, the property still need to be declared so it isn't as straightforward.